packages/odyssey-storybook/src/contributing/LabsComponents.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Contributing/Labs Components" />
+
+# Contibuting Odyssey Labs Components
+
+If you are wanting to add components to Odyssey Labs, read this doc first before creating a PR.
+
+## Requirements
+
+Getting components added to Odyssey requires a few steps to be completed first:
+
+1. Get approval from Odyssey design to see if that component even belongs in Odyssey.
```

Comment:
@jordankoschei-okta @edburyenegren-okta @magizh-okta 
Do we want to make it more clear that they need to pair with a designer either on their side or odysseys side to create the component first. That way the eng is not the only one figuring it out how things should look and work.

Or should it be okay to allow code contribution without design contribution attached to it?

Reply:
IMO if there are no approved designs, the component should be relegated to Labs until Design can do the full scope of our work. There should be an understanding that appearance and behavior are open to radical change if Design isn't involved initially.

Reply:
f0d2f1ca5aba96fa72b518750b03a2afd8b5499b